olive/passes/onnx/transformer_optimization.py

```diff
@@ -101,7 +101,7 @@ def _run_for_config(
         if config["optimization_options"]:
             self._set_fusion_options(run_config)
 
-        optimizer = transformers_optimizer.optimize_model(input=model.model_path, **run_config)
+        optimizer = transformers_optimizer.optimize_model(input=model.model_path, use_external_data_format=True, **run_config)
```

Comment:
need check ort version here since the flag doesn't exist with 1.15

Reply:
https://github.com/microsoft/Olive/blob/main/olive/systems/azureml/aml_pass_runner.py#L66-L76C1

need to be removed after upgrading to ORT 1.16

Reply:
Yes, maybe no hurry to add this changes now?

Reply:
@guotuofeng Thanks for reminding. Will add version check.

Reply:
@trajepl There are a couple of users who are trying large whisper models with multilingual with ORT nightly. We want to update Olive as well to support it. I can add a version check here!

Reply:
@xiaoyu-work , it seems the use_external_data_format flag is removed in PR https://github.com/microsoft/onnxruntime/pull/16912

Reply:
With latest nightly package, the transformer optimization will infer the flag of external data automatically.